### ISSUE TITLE
FavoriteMedia 1.6.0 - Favorite button on GIFs not working!

### ISSUE THREAD

**Issue by karimawii** [Dastan21/BDAddons#84]
Favorite button on GIFs doesn't add GIFs to Discord's default favorite GIFs tab.

**Comment by Dastan21** [comment]
Can you screen the star + tooltip of a message GIF pls ?

**Comment by karimawii** [comment]
![image](https://user-images.githubusercontent.com/69881381/170830237-d103b855-7b5a-43a8-9ddf-b5ea6e693332.png)

**Comment by Dastan21** [comment]
![image](https://user-images.githubusercontent.com/58090137/170830255-ff7a293d-15c8-423f-a606-edfd6d285456.png)
Are one of these enabled ?

**Comment by karimawii** [comment]
nope

**Comment by karimawii** [comment]
![image](https://user-images.githubusercontent.com/69881381/170830290-6689a722-9f48-449f-ab15-6a4cf744a83b.png)

**Comment by Dastan21** [comment]
Did you try restarting Discord ?

**Comment by karimawii** [comment]
yes I did, multiple times

**Comment by karimawii** [comment]
I also tried disabling the plugins, star the gif, then re-enable the plugin, it shows in the tab that way

**Comment by Dastan21** [comment]
Maybe toggle the "Replace GIF tab" option and disable it right after

**Comment by karimawii** [comment]
I did that, restarted, same issue

**Comment by Dastan21** [comment]
And if you favorite one then enable "Replace GIF tab" it is added in there ?

**Comment by karimawii** [comment]
nope
![image](https://user-images.githubusercontent.com/69881381/170831055-4003ebc6-beae-4fbf-9258-622bd3a23abf.png)

**Comment by karimawii** [comment]
I can't add any fav gifs to the replaced gifs tab

**Comment by Dastan21** [comment]
Anything on the console ? Ctrl+Shift+I to open it when this is enabled in BD Settings
![image](https://user-images.githubusercontent.com/58090137/170831086-5a6e65f2-cf2e-4dcc-9b4f-870b6b1ebfa1.png)

**Comment by karimawii** [comment]
no, but when I click the fav icon (with the replaced tab enabled), gif doesn't get added + when I switch channels/dms and go back the star icon is reset (not filled with color/outline only)

**Comment by Dastan21** [comment]
This is a bit tricky but I would like you to :
1. Open Devtool
2. Click on the icon top-left
3. Click on the star or "GIF button" top-right
4. Sharing what's on the "Elements" tab

**Comment by karimawii** [comment]
```
<div class="gifFavoriteButton-3ZomyF size-7nU0fO gifFavoriteButton-2IttKp fm-gifFavoriteButton" tabindex="-1" role="button"><svg class="icon-1m_PLf" aria-hidden="false" viewBox="0 0 24 24" width="16" height="16"><path fill="currentColor" d="M19.6,9l-4.2-0.4c-0.4,0-0.7-0.3-0.8-0.6l-1.6-3.9c-0.3-0.8-1.5-0.8-1.8,0L9.4,8.1C9.3,8.4,9,8.6,8.6,8.7L4.4,9 c-0.9,0.1-1.2,1.2-0.6,1.8L7,13.6c0.3,0.2,0.4,0.6,0.3,1l-1,4.1c-0.2,0.9,0.7,1.5,1.5,1.1l3.6-2.2c0.3-0.2,0.7-0.2,1,0l3.6,2.2 c0.8,0.5,1.7-0.2,1.5-1.1l-1-4.1c-0.1-0.4,0-0.7,0.3-1l3.2-2.8C20.9,10.2,20.5,9.1,19.6,9z M12,15.4l-3.8,2.3l1-4.3l-3.3-2.9 l4.4-0.4l1.7-4l1.7,4l4.4,0.4l-3.3,2.9l1,4.3L12,15.4z"></path></svg></div>
```

**Comment by Dastan21** [comment]
Is `useCustomGIFTab` set to true (when "Replace GIF tab" is disabled) in `FavoriteMedia.config.json` ?

**Comment by karimawii** [comment]
no it's false

**Comment by Dastan21** [comment]
Could you insert `console.log(propsDiv.className, type, this.settings.useCustomGIFTab)` at line 1861, open the devtool and go to a channel where there are GIFs then screenshot the results from the console pls

**Comment by karimawii** [comment]
like this?
![image](https://user-images.githubusercontent.com/69881381/170838651-193417e7-9124-4a48-84a5-494915f70019.png)

**Comment by Dastan21** [comment]
yup

**Comment by karimawii** [comment]
refreshing discord

**Comment by karimawii** [comment]
is this the one?
![image](https://user-images.githubusercontent.com/69881381/170838748-32e529ea-7353-4562-bb92-f43bdea580af.png)

**Comment by Dastan21** [comment]
Add `propsImg.src` in the console.log like this `console.log(propsDiv.className, type, this.settings.useCustomGIFTab)` and repeat pls

**Comment by karimawii** [comment]
![image](https://user-images.githubusercontent.com/69881381/170843791-cbbe887c-88b3-49d9-ad5b-a01ebd305720.png)

**Comment by Dastan21** [comment]
Okayyyyyyyyyyy I've finally found the bug, thanks you for all your time :) Will be fixed in the next minor update.

**Comment by Dastan21** [comment]
Should be fixed in v1.6.2.
